posts/2020-11-09-ocp-log-forwarding.adoc

```diff
@@ -368,7 +368,7 @@ Leave the other fields unchanged and submit. Copy the generated token value for
 
 == Integrating Open Liberty JSON Logs
 
-Liberty application pods output logs in JSON format, therefore it is recommended to set Fluentd to parse the JSON fields from the message body. To enable it, set the cluster logging instance's *managementState* field from *"Managed"* to *"Unmanaged"*. Note that once you set the value to *"Unmanaged"* any further changes to the _ClusterLogging_ or _LogForwarding_ instances will not be automatically detected. You will need to change it back to *"Managed"*.
```

Comment:
We should keep the last sentence somehow. If the user had any issues and were trying to fix it by modifying the other configurations, then their change will not take affect and unless they already know about it, _they'll be stuck indefinitely_

Reply:
This information is still in the doc, I just moved to be directly _after_ the example at line 386. I've restored the bold **Note** to call attention to it after the example.

The only reason I moved it was I felt it made the intro to the example easier to understand, since before it told the user to make a change but then told them they'd need to undo that change before they've even seen what it is. If it needs to go before the example I can make that edit but see what you think.